tests/extension_test.py

```diff
@@ -0,0 +1,12 @@
+"""Extension tests."""
+from sphinxcontrib.towncrier import _escape_version
+
+
+def test_escape_version():
+    """Test that escaping the version covers edge case.
+
+    RST substitution as the first item should be escaped.
+    """
+    assert _escape_version('|release|') == r'\ |release|'
+    assert _escape_version('|release') == '|release'
+    assert _escape_version('v|release|') == 'v|release|'
```

Comment:
These are 3 separate tests, they shouldn't appear as a single test in the pytest summary. Instead, use `@pytest.mark.parametrize` to declare different params for the same test function.